library/src/main/java/net/rdrei/android/dirchooser/DirectoryChooserFragment.java

```diff
@@ -330,20 +369,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     /**
      * Shows a confirmation dialog that asks the user if he wants to create a
-     * new folder.
+     * new folder. User can modify provided name, if it was not disallowed.
      */
     private void openNewFolderDialog() {
-        new AlertDialog.Builder(getActivity())
+        View dialogView = getActivity().getLayoutInflater().inflate(R.layout.dialog_new_folder, null);
```

Comment:
Parent argument is used by the inflater to determine and use appropriate LayoutParams when inflating. Then LayoutInflater is retrieved from the activity so it inherit the app's style.